vagrant/_commons.sh

```diff
@@ -90,7 +90,11 @@ function install_k8s {
             echo "download_container: false"
             echo "skip_downloads: false"
             } >> ./inventory/group_vars/all.yml
-            sed -i 's/^etcd_deployment_type: .*$/etcd_deployment_type: host/;s/^kubelet_deployment_type: .*$/kubelet_deployment_type: host/;s/^container_manager: .*$/container_manager: crio/' ./inventory/group_vars/k8s-cluster.yml
+            sed -i 's/^etcd_deployment_type: .*$/etcd_deployment_type: host/' ./inventory/group_vars/k8s-cluster.yml
+            sed -i 's/^kubelet_deployment_type: .*$/kubelet_deployment_type: host/' ./inventory/group_vars/k8s-cluster.yml
+            sed -i 's/^container_manager: .*$/container_manager: crio/' ./inventory/group_vars/k8s-cluster.yml
+            # TODO: https://github.com/kubernetes-sigs/kubespray/issues/4737
+            sed -i 's/^kube_version: .*$/kube_version: v1.13.5/' ./inventory/group_vars/k8s-cluster.yml
```

Comment:
The commit says moving to 1.14.1 but here you set it to 1.13.5?

Reply:
The 1.14.1 is the default version for docker, the v1.13.5 version is for CRI-O  given that can't coexist with Docker service. I can remove it once [this](https://github.com/kubernetes/kubernetes/pull/76505) is released for v1.15

Reply:
BTW, the idea of using +v1.14 is to consume the [kustomize](https://kubernetes.io/docs/concepts/overview/object-management-kubectl/kustomization/#) feature is it's integrated in kubectl